modules/ROOT/pages/database-administration/composite-databases/querying-composite-databases.adoc

```diff
@@ -17,7 +15,40 @@ Queries submitted to a Composite database may contain several `USE` clauses that
 
 Each constituent graph is named after the alias that introduces it into the Composite database.
 
-The examples assume the same setup as the one created in xref:composite-databases/administration.adoc[].
+[[composite-databases-queries-graph-set-up]]
+== Graph set-up
+
+The following set-up is required to recreate the below examples:
+
+.Create a standard database `movies2022`
+[source, cypher]
+----
+CREATE DATABASE movies2022
+----
+
+.Create a composite database `cineasts``
+[source, cypher]
+----
+CREATE COMPOSITE DATABASE cineasts
+----
+
+.Create database alias `cineasts`.`latest` for a local database in a composite database
+[source, cypher]
+----
+CREATE ALIAS `cineasts`.`latest`
+  FOR DATABASE movies2022
+----
+
+.Create remote database alias `cineasts`.`upcoming` in a composite database
+[source, cypher]
+----
+CREATE ALIAS `cineasts`.`upcoming`
+  FOR DATABASE upcoming
+  AT 'neo4j+s://other.dbms.com'
+  USER $user
+  PASSWORD $secretpassword
+----
+
 [[composite-databases-queries-single-graph]]
 === Query a single graph
```

Comment:
2 things for further down this page:

- [ ] Under `Limitations -> Graph accessing operations` it says `RETURN 1 + 2 AS number` is a graph accessing operation which I don't think it is. It should be updated to an actual graph accessing operation (so that there is still 2 examples)
- [ ] The `Cypher runtime` section says that `slotted` is the default runtime. I don't know if this is still correct or not. I assume someone in the composite team might know.

Reply:
For the runtime point - the default for EE is Pipelined, while it is Slotted for CE. There are other things to double check about this information (and a note to myself that this link will need to be updated when the new Runtimes chapter is available).
@JPryce-Aklundh